rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -4,12 +4,11 @@
 import sys
 import time
 
+sys.path.insert(1, os.path.dirname(__file__)) # noqa: E261
 from deoplete_jedi import cache, profiler, utils, worker
```

Comment:
Hmm, I don't know because don't use it until now.
But it related to deoplete core implementation.

/cc @Shougo 
If that works fine, what do you think about it?